### PR TITLE
Resolved fixed width issue of profile setting's left panel items.

### LINKED ISSUE #1794: Content is not adjusting when I resize the 2nd panel in Profile
https://user-images.githubusercontent.com/52249767/106749946-77869b00-6627-11eb-8d3e-427bf95c4625.mov

### PR DESCRIPTION
**Pull request by ace-smart**
Fixes #1794

### CODE REVIEW

File: ui/app/AppLayouts/Profile/LeftTab/Menu.qml [hunk 1]
```diff
@@ -28,6 +28,7 @@ ScrollView {
                 source: "../../../img/profile/" + modelData.filename
                 active: profileMenu.profileCurrentIndex === modelData.id
                 Layout.fillWidth: true
+                width: profileMenu.width
```
Comment (jrainville) [inline ui/app/AppLayouts/Profile/LeftTab/Menu.qml:31]: `parent.width` should do the trick as well I think

Comment (ace-smart) [reply ui/app/AppLayouts/Profile/LeftTab/Menu.qml:31]: I just checked your solution, but it is still keeping fixed width.
I am wondering `parent.width` will point `profileMenu.width` really.
```
ScrollView ->Column->Repeater->delegate:MenuButton
```

Comment (jrainville) [reply ui/app/AppLayouts/Profile/LeftTab/Menu.qml:31]: Ah right, the `Repeater` needs `width: parent.width` as well

Comment (ace-smart) [reply ui/app/AppLayouts/Profile/LeftTab/Menu.qml:31]: It is still not working, I am not sure the right reason.

**Review by jrainville — APPROVED** [review]

**Pull request merged**